packages/reqresp/src/messages/LightClientOptimisticUpdate.ts

```diff
@@ -15,5 +15,12 @@ export const LightClientOptimisticUpdate: ProtocolDefinitionGenerator<null, alta
     requestType: () => null,
     responseType: () => ssz.altair.LightClientOptimisticUpdate,
     contextBytes: getContextBytesLightclient((update) => modules.config.getForkName(update.signatureSlot), modules),
+    inboundRateLimits: {
+      /**
+       * Updates happens every block for the head. Can be optimized later on.
```

Comment:
having 1 / min in LightClientFinalityUpdate and 500 / min in LightClientOptimisticUpdate is way to disparate. The way to think about this numbers is = what is the max rate we can afford and what's the min the network needs to function? Such questions should be answered for each route, and justify the numbers